src/main/scala/reactor/core/scala/publisher/SFluxLike.scala

```diff
@@ -71,13 +75,11 @@ trait SFluxLike[+T] extends ScalaConverters {
     coreFlux.onErrorResume(f).asScala
   }
 
-  final def reduce[A](initial: A)(accumulator: (A, T) => A): SMono[A] = coreFlux.reduce[A](initial, accumulator).asScala
```

Comment:
So we are removing `reduce` and people should use `fold`?
This is breaking changes. Shouldn't we at least make it as deprecated first?

Reply:
yup. forward to fold and making it deprecate would be great. shall I do so?

Reply:
@jpork Yes please. I think that'd be prudent to do.

Reply:
@jpork I just realize that we don't have `reduce` without initial value here.

Reply:
Done.